kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -29,18 +23,18 @@ <h4 tabindex="-1">{{_('Support MDN, and it comes back to you.')}}</h4>
                 </p>
                 {% if is_popover %}
                     <div class="button-group">
-                        <a href="{{ faq_url }}" class="cta-link">
+                        <a href="{{url('payments')}}#contribute-faqs" class="cta-link">
                             {{_('Learn more about payments to MDN')}} {% include 'includes/icons/arrows/arrow.svg' %}
                         </a>
-                        <a href="{{ feedback_url }}" class="cta-link">
+                        <a href="{{url('payments')}}#contribute-feedback" class="cta-link">
                             {{_('Provide feedback')}} {% include 'includes/icons/arrows/arrow.svg' %}
                         </a>
                     </div>
                 {% endif %}
             </div>
         </div>
         <div class="column-half">
-            <form id="contribute-form" class="contribution-form" method="post" action="{{ payments_url }}" novalidate>
+            <form id="contribute-form" class="contribution-form" method="{% if recurring_payment and not request.user.is_authenticated %}get{%else%}post{%endif%}" action="{% if recurring_payment %}{{url('recurring_payment_subscription')}}{%else%}{{url('payments')}}{%endif%}" novalidate>
```

Comment:
The change from PR #5052 should be kept, and the ``url('recurring_payment_subscription')`` processed outside of the ``{% translation %}`` block.

Reply:
Done